src/test/java/com/radixdlt/crypto/HashTest.java

```diff
@@ -97,4 +158,16 @@ public void testHashValues512() {
 	private byte[] hash512(byte[] data) {
 		return Hash.hash512(data);
 	}
+
+	private Hash deadbeef() {
+		return new Hash(deadbeefString());
+	}
+
+	private String deadbeefString() {
+		return repeatString("deadbeef", 8);
```

Comment:
Guava has a `Strings.repeat(s, n)` if you want to use that instead.

Reply:
Oh great, I just found a Java >11. Great, will use that instead.